odl/discr/lp_discr.py

```diff
@@ -1022,10 +1007,8 @@ def uniform_discr_fromintv(interval, nsamples, exponent=2.0, interp='nearest',
 
     Parameters
     ----------
-    domain : `IntervalProd`
+    interval : `IntervalProd`
```

Comment:
Possible, but I wont touch that in this branch. This was simply harmonizing with the input parameter.